src/build_workflow/builders.py

```diff
@@ -5,15 +5,18 @@
 # compatible open source license.
 
 from abc import ABC
+from typing import Any
 
+from build_workflow.build_target import BuildTarget
+from build_workflow.builder import Builder
 from build_workflow.builder_from_dist import BuilderFromDist
 from build_workflow.builder_from_source import BuilderFromSource
 from manifests.input_manifest import InputComponentFromDist, InputComponentFromSource
 
 
 class Builders(ABC):
     @classmethod
-    def builder_from(self, component, target):
+    def builder_from(self, component: Any, target: 'BuildTarget') -> 'Builder':
```

Comment:
Both InputComponentFromDist and InputComponentFromSource are derived from InputComponent, can we use the base class instead of Any?